paper/paper.md

```diff
@@ -31,17 +31,40 @@ bibliography: paper.bib
+the _missing of the genome information and sub-sequence analysis on the annotated variants_. eVaiutilities _uses the output from the eVai_ and performs genome scale analysis of the eVai annotated 
+variants. It allows you to search across populations, variants and spanning information across multiple samples. It is encoded in RUST, which is a memory safe language and provides scalability and 
+uses RUST version: `rustc 1.89.0`.
 
 # Figures
 ![Interface of eVaiutilities](eVaiutilities.png)
 
 # eVaiutilities
 
-Variant identification and annotation across the human genome has wide biological significance and is of utmost importance taking into account the wide growth of the disease onset that are happening worldwide.  Many tools have been developed for the annotation of the variants that focuses on the annotation of the variants using the several pre-established database [@Wang2010-mh;@Obenchain2014-jj]. Apart from the significant development in the open source genomics [@McLaren2016-jo;@Pedersen2023-co], parallel efforts have been laid down for the variant identification and annotation in industrial research. eVai available through https://www.engenome.com/product/, provides customary analysis of variants by comparing the variants across the several tools and databases and provides TSV files with the annotations mapped. However, customary analysis is needed to make personal genomics easier. Additionally, for large scale variant searching, speed and scalability is needed. To address all these issues, eVaiutilities has been built using the RUST memory safe language features. eVaiutilities provides a compendium of options to allow for the searching of the variants across the population and generate customary analysis. It provides from variant report generation to the user specified database and sequence annotation. eVaiutilities features such as the transcript sequence to sequence spanning around the variants, allow the ease of the access to generate the molecular methods for the variants of interest and also allows for the categorical variant filtering. There are several features present in the eVaiutilities such as population wide analysis of a particular variant, coordinate search for variants across population, variant search across the specific coordinates with the specified variant across population, sequence specific analysis, reference and alternate allele search across the population. All the aforementioned features makes eVaiutilties an easy to use data management software for doing the customary analysis with the eVai. eVaiutilities is available from https://github.com/IBCHgenomic/eVaiutilities and is also available as a crate from https://crates.io/crates/eVaiutilities. 
+Variant identification and annotation across the human genome has wide biological significance and is of utmost importance taking into account the _wide growth of the disease onset that are 
```

Comment:
Please reword: "wide growth of the disease onset that are happening worldwide" - there has always been some disease in the world, I don't understand this sentence.